tasks/Task.cpp

```diff
@@ -93,13 +93,17 @@ bool Task::configureHook()
     cVb.buildFrom(cMb);
     task.set_cVe(cVb);
 
+    //set the time for trigger the timeout
+    timeout = base::Timeout(base::Time::fromSeconds(1));
```

Comment:
> the expected behavior is keep the vehicle in the position if no markers are detected.

I don't agree. If we want this, then the velocity/position controllers should start controlling at zero if it they have no commands.

If you have no feedback, you have no business controlling anything.